src/modules/report/plain.py

```diff
@@ -40,7 +40,9 @@ def get_report(self):
                     else:
                         output += "\nNo hunters were found"
         else:
-            print("\nKube Hunter couldn't find any clusters")
+            if vulnerabilities_len:
+                output += self.vulns_table()
+            output += "\nKube Hunter couldn't find any clusters"
```

Comment:
I have gone off the idea of saying "couldn't find any Kubernetes services" because in Kubernetes, a service is something different from what we mean here in kube-hunter. Maybe "kube-hunter couldn't find any Kubernetes APIs", wdyt?

Reply:
I agree,